src/main/java/haveric/recipeManager/tools/Version.java

```diff
@@ -52,6 +55,19 @@ private static boolean supports1_11() {
 
         return supports;
     }
+    private static boolean supports1_10() {

```

Comment:
There may be some things with Entity Types in 1.10 (specifically with FlagSummon), not seeing much else that updated in that version. I'm fine leaving it in if there's plans to use it